zipline/data/loader.py

```diff
@@ -218,11 +212,7 @@ def ensure_benchmark_data(symbol, first_date, last_date, now, trading_day,
     logger.info('Downloading benchmark data for {symbol!r}.', symbol=symbol)
 
     try:
-        data = get_benchmark_returns(
-            symbol,
-            first_date - trading_day,
-            last_date,
-        )
+        data = get_benchmark_returns(symbol)
```

Comment:
While I am not familiar with the codebase or charter of the project, I assume this function is used to get the benchmark returns for comparison against a backtest. Since the new function now only returns benchmark returns for the last 5 years, wouldn't this limit the ability of zipline-live to run backtests? Or are the users expected to run backtests against zipline and use zipline-live only for trading? I understand that zipline-live is not completely in sync with the zipline repository so that could be an issue.

Reply:
Yes, using IEX will limit backtests to the last 5 years.